### PR TITLE
[#47] [Chore] Set up CI workflow for testing a PR

### PR DESCRIPTION
**Pull request by markgravity**
https://github.com/markgravity/kmm-ic/issues/47

## What happened 👀

- Upgrade FastlaneRunner
- Create a test workflow to run tests in iOS Project and Shared module

## Insight 📝

n/a

## Proof Of Work 📹

CI successfully

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - name: Generate KMM frameworks for Cocoapods
+      run: |
+        cd ..
+        ./gradlew generateDummyFramework
+
+    - name: Install Pods Dependencies
+      run: bundle exec pod install
+
+    - name: Build and Test
+      run: bundle exec fastlane buildAndTest
```
Comment (phongvhd93) [inline .github/workflows/test.yml:58]: Will we post the code coverage result on the PR

Comment (markgravity) [reply .github/workflows/test.yml:58]: @phongvhd93 I will do it in this chore #53 🙏

**Comment by suho** [comment]
@markgravity You only implemented the CI part. Please update the title and description of this PR

**Comment by markgravity** [comment]
> @markgravity You only implemented the CI part. Please update the title and description of this PR

Updated

**Review by suho — APPROVED** [review]

**Review by phongvhd93 — APPROVED** [review]

**Pull request merged**